src/tools/kata-ctl/src/args.rs

```diff
@@ -93,3 +93,14 @@ pub enum IpTablesArguments {
     /// Configure iptables
     Metrics,
 }
+
+#[derive(Debug, Args, Error)]
+#[error("Argument is not valid")]
+pub struct ExecArguments {
+    #[clap(short, long, forbid_empty_values = true)]
+    /// pod sandbox ID or pouch container ID
```

Comment:
I think we do not have pouch container support in kata now (?

Reply:
Thanks, I will fix it later.